### ISSUE TITLE
Support Node v10

### ISSUE THREAD

**Issue by gajus** [DirtyHairy/node-libxl#19]
The installation is currently failing with two errors:

```
13 warnings generated.
  CXX(target) Release/obj.target/liblibxl/src/argument_helper.o
  CXX(target) Release/obj.target/liblibxl/src/util.o
../src/util.cc:51:45: error: no matching member function for call to 'NewInstance'
    Local<Value> newInstance = constructor->NewInstance(argc, argv);
                               ~~~~~~~~~~~~~^~~~~~~~~~~
/Users/gajus/.node-gyp/10.7.0/include/node/v8.h:3956:44: note: candidate function not viable: requires single argument 'context',
      but 2 arguments were provided
  V8_WARN_UNUSED_RESULT MaybeLocal<Object> NewInstance(
                                           ^
/Users/gajus/.node-gyp/10.7.0/include/node/v8.h:3953:44: note: candidate function not viable: requires 3 arguments, but 2 were
      provided
  V8_WARN_UNUSED_RESULT MaybeLocal<Object> NewInstance(
                                           ^
../src/util.cc:83:38: error: no matching member function for call to 'NewInstance'
    return scope.Escape(constructor->NewInstance(1, info));
                        ~~~~~~~~~~~~~^~~~~~~~~~~

```

**Comment by DirtyHairy** [comment]
I'll look into it.

**Comment by DirtyHairy** [comment]
Fixed in 0.3.1.
